src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -217,13 +217,24 @@ export const wrapWorkspace = ({ breadcrumbs, activeTab, title, showTabBar = true
 
     renderError() {
       const { workspaceError, errorText } = this.state
+      const groupURL = 'https://software.broadinstitute.org/firecloud/documentation/article?id=9553'
+      const authorizationURL = 'https://software.broadinstitute.org/firecloud/documentation/article?id=9524'
 
       return div({ style: { padding: '2rem' } }, [
         workspaceError.status === 404 ?
           h(Fragment, [
-            h2({}, ['Could not display workspace.']),
+            h2({}, ['Could not display workspace']),
             p({},
-              ['Either the requested workspace does not exist, or you do not have access. If you suspect you do not have access, please contact the workspace owner.'])
+              ['You are trying to access a workspace that either does not exist, or you do not have access to it.']),
+            p({}, [
```

Comment:
nit: The props object can be omitted if it's empty.

Reply:
Technically, but I thought we were generally making a point of including it for consistency? Like, if the child is just a string it doesn't have to be in an array, but we include that for the same reason.
Just looking to be consistent.